src/index.js

```diff
@@ -92,8 +86,7 @@ bot.on('message', (msg) => {
       return;
     }
   }
-
-  const response = commandHandler(player, enemy, inventory);
+  const response = commandHandler();
```

Comment:
I was tripped up with `commandHandler()` for a little bit. If you check my testing on Discord early in the night, I had removed the `()` and the commands were quite literally return the functions as strings. I see that adding the `()` will make the variable behave as a function, which is pretty cool!

Reply:
Heh yes functions can be stored in variables and then used, because functions are values much like any other value.